app/views/finders/_taxon_facet.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="app-taxonomy-select">
+  <%= render "govuk_publishing_components/components/select", {
+    id: 'level_one_taxon',
+    label: "Filter by topic",
+    options: taxon_facet.topics
+  }
+  %>
+  <div class="js-required govuk-form-group gem-c-select">
```

Comment:
Does this need a `js-hidden` class now too?

Reply:
Looks like there is some different behaviour clashing with js-hidden, so I'm not going to use that class.